components/PageButtons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gemeente-denhaag/pagebuttons",
-  "version": "0.1.1-alpha.2",
+  "version": "0.1.0",
```

Comment:
I'm guessing this is an accidental change which you meant to apply to the cta-image-content package.json. PageButtons is already released and being automatically versioned, we should not be changing the version for it manually.